src/scss/_sharing.scss

```diff
@@ -36,6 +36,17 @@
   }
 }
 
+#fullscreen-share-menu {
```

Comment:
Same for other things. This is not the right place to add this component and related styling.